mypy/main.py

```diff
@@ -552,8 +552,8 @@ def add_invertible_flag(flag: str,
         dest='shadow_file', action='append',
         help="When encountering SOURCE_FILE, read and type check "
              "the contents of SHADOW_FILE instead.")
-    add_invertible_flag('--fast-exit', default=False, help=argparse.SUPPRESS,
-                        group=internals_group)
+    add_invertible_flag('--fast-exit', default=True, help="makes mypy exit faster",
```

Comment:
I would add a bit more info here, explaining the down sides (for example files not being flushed at exit).